### PR TITLE
Daimo Design Refresh

### PR DESCRIPTION
**Pull request by pfista**
Draft PR to track progress on implementing new designs specified in #702

### CODE REVIEW

File: apps/daimo-web/src/components/PixelBackground.tsx [hunk 1]
```diff
@@ -0,0 +1,6929 @@
+export function PixelBackground(
```
Comment (dcposch) [inline apps/daimo-web/src/components/PixelBackground.tsx:1]: hmmmmm this will create a huge bundle size, right?

Comment (dcposch) [reply apps/daimo-web/src/components/PixelBackground.tsx:1]: recommend linking to asset image

File: apps/daimo-web/tailwind.config.js [hunk 2]
```diff
@@ -11,13 +11,24 @@ module.exports = {
       yellow: "#FFDC62",
       success: "#0CA01B",
       primary: "#007aff",
-      primaryLight: "#2888f7",
+      primaryLight: "#144B44",
```
Comment (dcposch) [inline apps/daimo-web/tailwind.config.js:14]: are we not using `primary` anywhere? otherwise it should change to green too

Comment (dcposch) [reply apps/daimo-web/tailwind.config.js:14]: all good, i can harmonize the color constants with the in-app ones in post

**Review by dcposch — APPROVED** [review]
LGreatTM

File: apps/daimo-web/src/app/globals.css [hunk 3]
```diff
@@ -15,6 +15,7 @@ html {
 body {
   overscroll-behavior-y: none;
   position: relative;
+  min-width: 430px;
```
[the remark this answers is not in the document]
Comment (pfista) [reply apps/daimo-web/src/app/globals.css:18]: Yes it should scale the viewport down below that size. Worth testing on the actual device to be sure though!

File: apps/daimo-web/src/components/PixelBackground.tsx (hunk 1 above)
Comment (pfista) [reply apps/daimo-web/src/components/PixelBackground.tsx]: I just pulled this out to be a static asset, though it only decreased first load js bundle size by 6kb

**Pull request merged**